webapp/interop_handler.go

```diff
@@ -36,6 +37,18 @@ func interopHandler(w http.ResponseWriter, r *http.Request) {
 		needsRedirect = true
 	}
 
+	q := r.URL.Query()
+	isMobileView, err := shared.ParseBooleanParam(q, "mobileView")
+	if err != nil {
+		http.Error(w, err.Error(), http.StatusBadRequest)
+		return
+	}
+	_, isValidMobileYear := validMobileYears[year];
+	if isMobileView != nil && !isValidMobileYear {
+		year = defaultRedirectYear
+		needsRedirect = true
```

Comment:
minor detail: should it redirect to interop-2024? or interop-2024?mobile-view?

Reply:
I don't think that is the case testing on the staging deployment - https://2024-08-25-mobile-resu-dot-wptdashboard-staging.uk.r.appspot.com/interop-2023?mobile-view

Reply:
nm it worked! The staging deployment was not up-to-date, https://2024-08-25-mobile-resu-dot-wptdashboard-staging.uk.r.appspot.com/interop-2023?mobile-view. Thanks!